lib/ui/myplot/PlotListItem.dart

```diff
@@ -119,9 +119,14 @@ class PlotListItem {
                       children: <Widget>[
                         _buildMainTextView(viewModel, itemStyle),
                         SizedBox(width: itemStyle.imageLineSpace),
-                        //FIXME: This value parameters are hardcoded right now, later should be the current stage number divided by all the stages.
-                        CircularProgress(viewModel: CircularProgressViewModel(
-                            initialValue: 20, content: _buildPlotImageContent(itemStyle, viewModel.imageUrl)))
+                        Stack(
+                          alignment: AlignmentDirectional.center,
+                          children: <Widget>[
+                            _buildPlotImage(viewModel.imageUrl, itemStyle),
+                            //FIXME: This value parameters are hardcoded right now, later should be the current stage number divided by all the stages.
+                            CircularProgress(progress: 80),
```

Comment:
grab the value from the viewModel named  "progress"  double 0.0 - 1.0

Reply:
Fixed